content/400-reference/200-api-reference/100-prisma-schema-reference.mdx

```diff
@@ -1066,6 +1068,33 @@ model User {
 </tab>
 </TabbedContent>
 
+##### Define a scalar list with a default value
+
+<TabbedContent tabs={[<FileWithIcon text="Relational databases" icon="database"/>, <FileWithIcon text="MongoDB" icon="database"/>]}>
+<tab>
+
+```prisma highlight=3;normal
+model User {
+  id             Int      @id @default(autoincrement())
+  posts          Post[]
+  favoriteColors String[] @default(["red", "blue", "green"])
+}
+```
+
+</tab>
+<tab>
+
+```prisma highlight=3;normal
+model User {
+  id             String   @id @default(auto()) @map("_id") @db.ObjectId
+  posts          Post[]
+  favoriteColors String[] @default(["red", "blue", "green"])
+}
+```
```

Comment:
Default is actually in line 4, so that needs highlighting:
````suggestion
```prisma highlight=4;normal
model User {
  id             Int      @id @default(autoincrement())
  posts          Post[]
  favoriteColors String[] @default(["red", "blue", "green"])
}
```

</tab>
<tab>

```prisma highlight=4;normal
model User {
  id             String   @id @default(auto()) @map("_id") @db.ObjectId
  posts          Post[]
  favoriteColors String[] @default(["red", "blue", "green"])
}
```
````